### DISCUSSION TITLE
Is there way to disable an entry but not hide the text?

### DISCUSSION THREAD

**Discussion by JP441** [TomSchimansky/CustomTkinter#1740] [Q&A]
I have an entry box which I do not want the user to be able to type in however, I want the text to still be displayed. I have tried to disable it but when I do that the text disappears. Sorry for the noob question but I'm not sure how to achieve this. Thanks for your help :)

![entrys with numbers](https://github.com/TomSchimansky/CustomTkinter/assets/119985826/866ce12c-5b28-4f69-ae8d-f10022c8c38f)

state='normal' ^^^

![entry without numbers](https://github.com/TomSchimansky/CustomTkinter/assets/119985826/419c3bc5-be37-4ae9-b4f8-99c93db040d9)

state='disabled' ^^^

**Comment by Akascape** [comment]
@JP441 
Switch to `state='readonly'`

**Reply by JP441** [reply]
Thank you that worked!
